sound/soc/sof/intel/Kconfig

```diff
@@ -249,6 +249,15 @@ config SND_SOC_SOF_HDA_PROBES
 	  Say Y if you want to enable probes.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_HDA_DISABLE_PAUSE
```

Comment:
can we use DMI_L1 directly instead of adding another kconfig? if ALWAYS_DMI_L1 is enabled, we should automatically disable pause no?

Reply:
I did it the other way around.

a) you may want to disable PAUSE if you feel like it for tests.
b) you may want to explore what happens when DMI L1 is enabled, even without pause
c) if you disable pause, the code automatically selects DMI L1 since it's safe to do so.